baselines/fedpac/README.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Personalized Federated Learning with Feature Alignment and Classifier Collaboration
+url: https://openreview.net/forum?id=SXZr8aDKia
+labels: [label1, label2] # please add between 4 and 10 single-word (maybe two-words) labels (e.g. "system heterogeneity", "image classification", "asynchronous", "weight sharing", "cross-silo")
+dataset: [dataset1, dataset2] # list of datasets you include in your baseline
```

Comment:
Don't forget to add meaningful labels for your baseline. Then remove the comments.